uhabits-android/src/main/java/org/isoron/uhabits/activities/common/dialogs/FrequencyPickerDialog.kt

```diff
@@ -22,97 +22,101 @@ package org.isoron.uhabits.activities.common.dialogs
 import android.app.Dialog
 import android.os.Bundle
 import android.view.LayoutInflater
-import android.view.View
 import android.widget.EditText
 import android.widget.LinearLayout
 import android.widget.RadioButton
 import android.widget.TextView
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatDialogFragment
-import kotlinx.android.synthetic.main.frequency_picker_dialog.view.*
 import org.isoron.uhabits.R
+import org.isoron.uhabits.databinding.FrequencyPickerDialogBinding
 
 class FrequencyPickerDialog(
     var freqNumerator: Int,
     var freqDenominator: Int
 ) : AppCompatDialogFragment() {
+    private var _binding: FrequencyPickerDialogBinding? = null
+    private val binding get() = _binding!!
```

Comment:
Could this be replaced by `private lateinit var binding: FrequencyPickerDialogBinding` ?

Reply:
From https://developer.android.com/topic/libraries/view-binding#fragments, the example is different from the one for activities, I think because of this note further down:

"Note: Fragments outlive their views. Make sure you clean up any references to the binding class instance in the fragment's [onDestroyView()](https://developer.android.com/reference/kotlin/androidx/fragment/app/Fragment#ondestroyview) method."

And `FrequencyPickerDialoger` is a fragment to this should apply to it. So we should be able to assign to `binding` (`_binding` in the current) twice. Or am I missing something?